yaml/ruleset.go

```diff
@@ -137,12 +137,24 @@ func (r *Rules) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err == nil {
 		r.Branch = rules.Branch
 		r.Comment = rules.Comment
-		r.Event = rules.Event
 		r.Path = rules.Path
 		r.Repo = rules.Repo
 		r.Status = rules.Status
 		r.Tag = rules.Tag
 		r.Target = rules.Target
+
+		// account for users who use non-scoped pull_request event
+		events := []string{}
+
+		for _, e := range rules.Event {
+			if e == "pull_request" {
```

Comment:
I prefer to utilize [strings.EqualFold](https://pkg.go.dev/strings#EqualFold) for comparison. 

```suggestion
			if strings.EqualFold(e, "pull_request")  {
```

Reply:
Sounds good! I'll change that